bpf-gpl/nat.h

```diff
@@ -118,9 +133,17 @@ static CALI_BPF_INLINE struct calico_nat_dest* calico_v4_nat_lookup2(__be32 ip_s
 								     __be32 ip_dst,
 								     __u8 ip_proto,
 								     __u16 dport,
-								     bool from_tun)
+								     bool from_tun,
+								     __u8 *drop)
 {
 	struct calico_nat_v4_key nat_key = {
+		.prefixlen = 88,
```

Comment:
This has to be (sizeof(struct calico_nat_v4_key) - sizeof(uint32_t) - sizeof(nat_key.pad))*8
I feel this as well has to be kept in sync

Reply:
I was thinking the same as Shaun, took me a while to see where the 88 comes from. I'd define a marco, something like `NAT_KEY_WITH_SRC_MATCH`, whatever makes sense so that the code is readable right away.